intel_extension_for_transformers/neural_chat/models/model_utils.py

```diff
@@ -1029,6 +1029,11 @@ def predict_stream(**params):
             However, your messages resulted in {input_token_len} tokens. Please reduce the length of the messages.",
         )
         set_latest_error(ErrorCodes.WARNING_INPUT_EXCEED_MAX_SEQ_LENGTH)
+        ret = {
```

Comment:
I think it's better to yield at upper layer. It will cause many function issues if change at here.

Reply:
@lvliang-intel what do you suggest?